jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4291,8 +4292,17 @@ ConstantPool getConstantPool() {
  * @return		true if class is a nestmate of this class; false otherwise.
  *
  */
-public boolean isInSameNest(Class<?> that) {
+public boolean isNestmateOf(Class<?> that) {
```

Comment:
The new name isn't consistent with the (old) behavior.

Reply:
From what I understand, this method should return true if the two classes have the same nest host (ie. are in the same nest). In both cases, it would compare the nest host of the two classes and return true if they have the same nest host and false otherwise. 

The spec gives:
```
public boolean isNestmateOf​(Class<?> c)

Determines if the given Class is a nestmate of the object represented by this Class. Two classes are nestmates if they have the same nest host.

Parameters:
    c - the class to check
Returns:
    true if this class and c are valid members of the same nest; and false otherwise.
```
What is the inconsistency between the name and the behaviour?

Reply:
Consider classes A and B nested within C: The way I originally read the new name I would expect `A.class.isNestmateOf(B.class)` to return false - I think the old name was clearer, but I understand you have to respect the specification.